Library/Homebrew/test.rb

```diff
@@ -17,6 +17,15 @@
   ENV.extend(Stdenv)
   ENV.setup_build_environment
 
+  # This ensures all executables in test do are called with a full path, i.e.
+  # bin/"wget", or Formula["openssl"].opt_bin/"openssl" rather than just "wget".
+  # It's very likely to break tests, so phase it in via CI rather than user
```

Comment:
I'd rather we have consistent testing. This is something we have to correct contributors on all the time, and I'm pretty sick of telling people to fix it, and I'm pretty sure people are sick of me telling them to fix it.

Whilst technically _"You only absolutely need to do x if x is `keg_only` or x tool you're trying to use from elsewhere in the test is `keg_only`"_ could be correct, that's a hell of a lot more complicated to explain to people & I don't see the upside over this method.